newsletter-december-2022.rst

```diff
@@ -0,0 +1,82 @@
+.. post:: Dec 1, 2022
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - December 2022
+========================================
+
+Here are the first features and updates that have hatched since we announced a Q4 focus on core platform features in the :doc:`previous newsletter </newsletter-december-2022>`.
+
+
+New features
+------------
+
+The latest updates from our team:
+
+- Follow us in the Fediverse (Mastodon): `@readthedocs@fosstodon.org <https://fosstodon.org/@readthedocs>`_
+- The *entire* build process can now be overridden (if you need it). :doc:`See the Announcement </build-customization>`
+  - ...this is nice if you want to use a framework that isn't directly supported. We wrote some examples :ref:`Pelican <readthedocs:build-customization:pelican>` and :ref:`Docsify <readthedocs:build-customization:Docsify>`.
+  - ...but you can also *extend* the build process for instance you can :ref:`skip a build <readthedocs:build-customization:cancel-build-based-on-a-condition>`!
+- We are getting ready for Sphinx 6, almost there! (scroll down for details)
+- :doc:`Server-Side Search API v3 <readthedocs:server-side-search/index>` has been released.
+- We organized a larger refactor of our user documentation to comply with the `Diátaxis methodology framework <https://diataxis.fr>`_. So far, we broke it down into `73 tasks and counting <https://github.com/readthedocs/readthedocs.org/issues?q=is%3Aissue++diataxis+iteration+>`_. The tasks are big and small.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+
+Upcoming features
+-----------------
+
+(from last newsletter)
+
+- A 1.2.0 release of :ref:`sphinx-rtd-theme <sphinx_rtd_theme110_upcoming_releases>` will support docutils 0.18 and Sphinx 6.
+- We're working on improving our integration with Material for MkDocs, which is a great theme for MkDocs documentation projects.
+- Many improvements to our URL handling code, which will allow us to support more flexible URL configurations for projects.
+- A search redesign to make it nicer across our dashboard and in-doc search experiences. 
+- 404 pages are being improved by contextualization the user message, giving relevant guidance to readers and project owners.
+
+
+Possible issues
+---------------
```

Comment:
probably mention https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-98pf-gfh3-x3mp in this section

Reply:
Yes, should definitely be mentioned! Thanks!

I adapted the "New features" section to accommodate fixes like this. It's to avoid having security fixes in "Possible issues", it indicates they aren't really fixed :)

It's now added further up.